pkg/scheduler/core/spreadconstraint/util.go

```diff
@@ -32,6 +32,7 @@ func sortClusters(infos []ClusterDetailInfo) {
 			return infos[i].Score > infos[j].Score
 		}
 
-		return infos[i].Name < infos[j].Name
+		// It makes more sense to compare the available replicas of the cluster.
+		return infos[i].AvailableReplicas > infos[j].AvailableReplicas
```

Comment:
If the AvailableReplicas is the same, I prefer to keep it in name order, or it's unpredictable.

Reply:
Yeah, I will adjust it.